hardhat.config.js

```diff
@@ -10,6 +10,7 @@ module.exports = {
     hardhat: {
       forking: {
         url: "https://eth-mainnet.alchemyapi.io/v2/7dW8KCqWwKa1vdaitq-SxmKfxWZ4yPG6",
+        blockNumber: 14390000
```

Comment:
see more: [why need to pin a block](https://hardhat.org/hardhat-network/docs/guides/forking-other-networks#pinning-a-block).

> to set up a test suite that depends on forking we recommend forking from a specific block number.

Reply:
sometimes it failed: https://github.com/zhyd1997/usingtellor/actions/runs/3060543371/attempts/1

while sometimes it worked: https://github.com/zhyd1997/usingtellor/actions/runs/3060543371

need to **retry** if it failed.

Reply:
@themandalore PTAL ☝🏿